RELEASE_NOTES.rst

```diff
@@ -1,3 +1,68 @@
+coala-bears 0.8.0
+=================
+
+For this release, we have had 19 contributors from around the world
+contributing about 176 commits to just coala-bears in the past 9 weeks.
+
+Here are the important changes or additions:
+
+**New Bears**
+
+- ``CSecurityBear`` - Lints C/C++ files and identifies possible security
+  issues.
+
+- ``HappinessLintBear`` - Checks JavaScript files for semantic and syntax
+  errors using ``happiness``.
+
+- ``WriteGoodLintBear`` - Lints the text files using ``write-good`` for
+  improving proses.
+
+- ``coalaBear`` - Check for the correct spelling and casing of ``coala``
+  in the text files.
+
+- ``VultureBear`` - Checks Python code for unused variables and functions
+  using ``vulture``.
```

Comment:
@hypothesist just to confirm, did you go through the commit log? It'd be shame to forget something here with all the great stuff that happened.